examples/react-navigation/src/components/modals/AlwaysOpen.js

```diff
@@ -21,6 +21,7 @@ export class AlwaysOpen extends React.PureComponent {
         ref={this.modal}
         style={s.content__modal}
         alwaysOpen={85}
+        modalHeight={350}
```

Comment:
I think it would be better to have this example on the CustomStyle modal. Also, don't forget to add this example on expo/react-native-navigation also

Reply:
Working on it